frontend/public/components/dashboard/capacity-card/capacity-item.tsx

```diff
@@ -6,7 +6,7 @@ import { GaugeChart } from '../../graphs/gauge';
 const NOT_AVAILABLE = 'Not available';
 
 export const CapacityItem: React.FC<CapacityItemProps> = React.memo(({ title, used, total, formatValue, isLoading = false }) => {
-  const error = used == null || total == null;
+  const error = (!used || !total) && 'No Data';
```

Comment:
Do we need to differentiate between 0 and null/undefined?

Better if this is always a string instead of string | boolean

Reply:
How about this?

```suggestion
  const error =  _.isFinite(total) && _.isFinite() ? '' : 'No Data';
```

Reply:
That works 👍